### ISSUE TITLE
Process gets killed on remote host

### ISSUE THREAD

**Issue by danteblades** [quasar/Quasar#907]
I have a remote host which I did a reinstall of windows 10 (running the latest build) for almost 2 years there were no issues but now after the reinstall it seems that my remote desktop windows just randomly will crash and never connect back. I have to re-rerun the built client to get the host to come back online. However, if I don't have any window open the server will not disconnect. Only the ones which have an active window open such as file transfers or a remote desktop.

 Is there any way to debug this and find out what is causing it to drop? I can run 3 servers on the same host and all 3 will drop at the exact same time. I'm running 1.3.0.0 and 1.4.0.0. Nothing changed besides reinstalling windows on the host and then running the same server which was running before. If this isn't clear or doesn't make sense please let me know.

**Comment by MaxXor** [comment]
Please use the bug report template: https://github.com/quasar/Quasar/issues/new?template=bug_report.md

It is not clear for me what steps are necessary to reproduce this issue. Do you mean open application windows like firefox which crash the remote desktop or do you mean Quasar windows like file manager? Please provide exact steps to reproduce this.

**Comment by danteblades** [comment]
I have a client
Previously I had 2 servers running on that client, one for each desktop to monitor both at the same time. This worked for almost 3 years without an issue.
I needed to reinstall Windows 10 on that client.

After reinstalling, whenever I have any active windows open from that client (For example, remote desktop, or file manager, but others as well) The server will crash on the client's end and I have to restart the server. 

To test I am now running 3 servers on that client. I only use 2 to display each monitor at all times. The two with active remote desktops open will crash, but the 3rd one is always running. 

I have already tried running only 1, still crashes. 

This is happening on 1.3.0.0 and 1.4.0.0 both release builds. Nothing has changed on my end, besides a Windows reinstall. It is also running over LAN so internet disconnection isn't an issue. It will also happen at random. It can run fine for 12+ hours, but then also crash 5 minutes after relaunching the server again. Could it be that there something in the latest Windows 10 build (2004) that is killing active connections?

Sorry if this doesn't make sense. It was working perfectly fine and stable before so it's confusing to me what could be causing the issue.

**Comment by BurntDog** [comment]
Make sure you're all up to date and recent .Net is installed. Remember a fully updated windows 10 Defender will automatically restart itself after a while. Did you make a exclusion?

**Comment by danteblades** [comment]
So I used group policy editor to turn off windows defender, as well as switching it off in settings, but for some reason the quick scan was still running at random times and killing the process? They certainly made defender very resilient this time around. I'm 99% sure that resolved the problem. I added it to exclusions just to be sure. 

Sorry for the trouble. Thanks for the help. I thought the group policy editor would disable it completely but it seems the quick scan still runs regardless. I knew they made it harder to disable defender but damn!
